Bottom: 95c22ab7173dd7213ed18f227fd192160b79404d
Top:    ad64ccd5173e57395292df0b688640015da321ee
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-30 19:54:39 -0400

Refresh of spi

---

diff --git a/drivers/spi/Makefile b/drivers/spi/Makefile
index 1ac2deb..f8cdf79 100644
--- a/drivers/spi/Makefile
+++ b/drivers/spi/Makefile
@@ -31,7 +31,7 @@ obj-$(CONFIG_SPI_FSL_SPI)		+= spi-fsl-spi.o
 obj-$(CONFIG_SPI_GPIO)			+= spi-gpio.o
 obj-$(CONFIG_SPI_IMX)			+= spi-imx.o
 obj-$(CONFIG_SPI_LM70_LLP)		+= spi-lm70llp.o
-obj-$(CONFIG_SPI_LPC313X)		+= spi_lpc313x.o
+obj-$(CONFIG_SPI_LPC313X)		+= spi-lpc313x.o
 obj-$(CONFIG_SPI_MPC512x_PSC)		+= spi-mpc512x-psc.o
 obj-$(CONFIG_SPI_MPC52xx_PSC)		+= spi-mpc52xx-psc.o
 obj-$(CONFIG_SPI_MPC52xx)		+= spi-mpc52xx.o
diff --git a/drivers/spi/spi_lpc313x.c b/drivers/spi/spi-lpc313x.c
similarity index 100%
rename from drivers/spi/spi_lpc313x.c
rename to drivers/spi/spi-lpc313x.c
